src/reference/sectionB_elements.inc

```diff
@@ -96,7 +96,7 @@ An :code:`import units` element information item (referred to in this specificat
 
       The value of the :code:`name` attribute MUST be a :ref:`CellML identifier<specA_cellml_identifier>`.
 
-      The value of the :code:`name` attribute MUST NOT be identical to the value of the :code:`name` attribute of any other :code:`units` or :code:`import units` element in the :ref:`CellML infoset<specA_cellml_infoset>`.
+      The value of the :code:`name` attribute MUST NOT be identical to the :code:`name` attribute of any other :code:`units` or :code:`import units` element in the :ref:`CellML infoset<specA_cellml_infoset>`.
```

Comment:
Does this `value of the` not stay?

Reply:
and similar ones that are identical to

Reply:
@MichaelClerx and @kerimoyle may comment on this, but I understand that "value" was mistakenly used and was therefore reverted.

Reply:
Yeah I'd agree with that

Reply:
I think it was mistakenly used in some places but not in this circumstance.

Reply:
Fixed